### ISSUE TITLE
Intellisense doesn't work.

### ISSUE THREAD

**Issue by ttSpace** [microsoft/PTVS#6895]
**Describe the bug**

![image](https://user-images.githubusercontent.com/82852072/155933700-0a869023-513d-4573-8f15-753ee5e8d676.png)

**Steps to Reproduce**
1. Create a Python Application.
2. Type ```import```
(FYI: https://docs.microsoft.com/en-us/visualstudio/python/editing-python-code-in-visual-studio?view=vs-2022 )

**Expected behavior**
 
As you type, your completion list should open and prompt for your input accordingly. 

**Additional context and screenshots**

Nothing came up.

![image](https://user-images.githubusercontent.com/82852072/155934887-32c45b6a-a131-4d2d-8017-f9f442f9bcb1.png)

**Comment by AdamYoblick** [comment]
Looks like autocomplete is completely broken in the latest version of PTVS with latest pylance. I've confirmed that this isn't a pylance issue, since the same version of pylance works in VSCode. Also, nothing has changed in PTVS that would have broken this, so if I had to guess, I'd say something in VS changed - either LSP related or intellicode/intellisense related and we need to respond to the change, maybe a package update or something similar hopefully.

I will reach out to the LSP team and go from there. 😄
